src/wp-includes/option.php

```diff
@@ -3214,3 +3214,126 @@ function wp_autoload_values_to_autoload() {
 
 	return array_intersect( $filtered_values, $autoload_values );
 }
+
+
+/**
+ * Filters site metadata for multisite installations.
+ *
+ * This function is specific to multisite setups and provides a way to retrieve
+ * network options. It enforces parameters' constraints and suggests using an alternative function.
+ *
+ * @param mixed $current The value to return if no changes are made.
+ * @param int $object_id Site ID.
+ * @param string $meta_key Metadata key.
+ * @param bool $single Whether to retrieve a single value. Must be true.
+ *
+ * @return mixed The network option value for the specified meta key, or false if the function is used incorrectly.
+ */
+function filter_get_site_metadata( $current, $object_id, $meta_key, $single ) {
+	if ( ! is_multisite() ) {
+		_doing_it_wrong( 'get_metadata', __( 'This function is only available in multisite installations.' ), '6.8.0' );
```

Comment:
Shouldn't it, judging the function name, be something like the following:

```
_doing_it_wrong( 'filter_get_site_metadata', __( 'This function is only available in multisite installations.' ), '6.8.0' );
```

or 

```
_doing_it_wrong( __FUNCTION__, __( 'This function is only available in multisite installations.' ), '6.8.0' );
```